go.mod

```diff
@@ -0,0 +1,6 @@
+module manifest-tool
+
+go 1.14
+
+require (
+)
```

Comment:
If you instead create this file via `go mod init`, it will convert `vendor.conf` automatically:

```console
$ go-mod-env go mod init github.com/estesp/manifest-tool
go: creating new go.mod: module github.com/estesp/manifest-tool
go: copying requirements from vendor.conf

$ cat go.mod
module github.com/estesp/manifest-tool

go 1.15

require (
	github.com/Azure/go-ansiterm v0.0.0-20170929234023-d6e3b3328b78
	github.com/Microsoft/go-winio v0.4.14
	github.com/Microsoft/hcsshim v0.8.7-0.20190417211021-672e52e9209d
	github.com/beorn7/perks v0.0.0-20190414131216-e7f67b54abbe
	github.com/codegangsta/cli v1.2.0
	github.com/containerd/containerd v1.2.2
	github.com/containerd/continuity v0.0.0-20181203112020-004b46473808
	github.com/docker/cli v0.0.0-20191023165302-c2ae869f6fc4
	github.com/docker/distribution v2.7.1-0.20190205005809-0d3efadf0154+incompatible
	github.com/docker/docker v17.12.0-ce-rc1.0.20200116195852-71e07f91307a+incompatible
	github.com/docker/docker-credential-helpers v0.6.1
	github.com/docker/go-connections v0.4.0
	github.com/docker/go-metrics v0.0.0-20170502235133-d466d4f6fd96
	github.com/docker/go-units v0.4.0
	github.com/docker/libtrust v0.0.0-20150526203908-9cbd2a1374f4
	github.com/go-yaml/yaml v0.0.0-20200602174213-b893565b90ca
	github.com/gogo/protobuf v1.0.0
	github.com/golang/protobuf v1.1.0
	github.com/gorilla/mux v1.8.0
	github.com/konsorten/go-windows-terminal-sequences v1.0.2
	github.com/mattn/go-shellwords v1.0.10
	github.com/matttproud/golang_protobuf_extensions v1.0.1
	github.com/morikuni/aec v1.0.0
	github.com/opencontainers/go-digest v1.0.0-rc1
	github.com/opencontainers/image-spec v1.0.1
	github.com/opencontainers/runc v1.0.0-rc8.0.20190926000215-3e425f80a8c9
	github.com/opencontainers/runtime-spec v1.0.2-0.20190207185410-29686dbc5559
	github.com/pkg/errors v0.8.1
	github.com/prometheus/client_golang v0.8.0
	github.com/prometheus/client_model v0.0.0-20170216185247-6f3806018612
	github.com/prometheus/common v0.0.0-20180518154759-7600349dcfe1
	github.com/prometheus/procfs v0.0.0-20180612222113-7d6f385de8be
	github.com/sirupsen/logrus v1.4.1
	github.com/vbatts/tar-split v0.11.1
	golang.org/x/crypto v0.0.0-20190411191339-88737f569e3a
	golang.org/x/net v0.0.0-20190522155817-f3200d17e092
	golang.org/x/sync v0.0.0-20190227155943-e225da77a7e6
	golang.org/x/sys v0.0.0-20190602015325-4c4f7f33c9ed
	golang.org/x/time v0.0.0-20180412165947-fbb02b2291d2
	google.golang.org/genproto v0.0.0-20170523043604-d80a6e20e776
	google.golang.org/grpc v1.12.0
)
```

This will probably need additional cleanup/work, but it at least gives a more "correct" starting point for the conversion.

Reply:
Thanks @tianon; I hadn't had much time to respond to this, but yes, I already started this work on the `v2-rework` branch and had thought about "deprecating" the current main branch/v1.0.x and not trying to make it work properly with `go mod`; but a lot of things have gotten in the way of finishing my "v2" parity with function in v1.0.x, although I think it is really close. Here is the v2-rework go.mod: https://github.com/estesp/manifest-tool/blob/v2-rework/go.mod